packages/fluentbit/extra/start_fluentbit.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -o errexit
+set -o xtrace
+set -o nounset
+
+# Start fluentbit
+exec /opt/mesosphere/bin/fluent-bit "$@"
```

Comment:
Are we going to need to put anything in this startup script? I wonder if we can do without it.

Reply:
I believe you're right.